docs/data/charts/radar/radar.md

```diff
@@ -72,6 +72,19 @@ With composition you can add this behavior with the `<RadarAxisHighlight axisHig
 
 {{"demo": "DemoRadarAxisHighlight.js" }}
 
+### Series Highlight
+
+To set the highlight on series, use the `highlight` prop with `'series'` value.
+This highlight can be controlled with `highlightedItem` value and `onHighlightChange` callback.
+
+With composition you can pass those props to the `RadarDataProvider`.
+
+{{"demo": "DemoRadarSeriesHighlight.js" }}
```

Comment:
Would it make sense to explain what defines the order in which series are highlighted? 

I suppose it's the same order as the legend, but I was slightly confused at first

https://github.com/user-attachments/assets/9340c133-ab68-4de7-81dd-8aea044ad6a3

Reply:
I tried an explaination, but not 100% convinced